### ISSUE TITLE
Error deploying to Web App

### ISSUE THREAD

**Issue by NenadCrncec** [microsoft/vscode-azureappservice#2411]
Does this occur consistently? Yes
Repro steps:
Project is javascript/typescript based app - single-spa root app, that is built using npm and node.
1. Rightclick on project folder and select "Deploy to Web App"
2. Select existing subscription, resource group and web app
3. Error appears

Action: appService.Deploy
Error type: TypeError
Error Message: Cannot read properties of undefined (reading 'createClient')

Version: 0.24.7
OS: darwin
OS Release: 22.1.0
Product: Visual Studio Code
Product Version: 1.76.0
Language: en

<details>
<summary>Call Stack</summary>

```
extension.bundle.js:2:3774915
fulfilled extension.bundle.js:2:3773586
```

</details>

**Comment by alexweininger** [comment]
As a workaround, try right-clicking the web app you want to deploy to, and deploying that way. I think this bug is specific to right-click folder entry point.

**Comment by alexweininger** [comment]
This issue has been fixed by the [0.7.0 release of the Azure Resources extension](https://github.com/microsoft/vscode-azureresourcegroups/releases/tag/v0.7.0). Please update the Azure Resources extension to version 0.7.0.
